driver/esp32/ili9XXX.py

```diff
@@ -666,7 +679,82 @@ def __init__(self,
             {'cmd': 0x11, 'data': bytes([0]), 'delay': 20},
             {'cmd': 0x29, 'data': bytes([0]), 'delay': 120}
         ]
-        
+
         super().__init__(miso, mosi, clk, cs, dc, rst, power, backlight, backlight_on, power_on,
             spihost, mhz, factor, hybrid, width, height, self.colormode, rot, invert, double_buffer, half_duplex, display_type=self.display_type,
             asynchronous=asynchronous, initialize=initialize)
+
+class st7789(ili9XXX):
+    PORTRAIT = const(0)
+    LANDSCAPE = const(1)
+    INVERSE_PORTRAIT = const(2)
+    INVERSE_LANDSCAPE = const(3)
+
+    def __init__(self,
+        miso=-1, mosi=19, clk=18, cs=5, dc=16, rst=23, power=-1, backlight=4, backlight_on=1, power_on=0,
+        spihost=esp.HSPI_HOST, mhz=40, factor=4, hybrid=True, width=135, height=240, start_x=None, start_y=None,
+        colormode=COLOR_MODE_BGR, rot=PORTRAIT, invert=True, double_buffer=True, half_duplex=True,
+        asynchronous=False, initialize=True):
+
+        # Make sure Micropython was built such that color won't require processing before DMA
+
+        if lv.color_t.__SIZE__ != 2:
+            raise RuntimeError('st7789 micropython driver requires defining LV_COLOR_DEPTH=16')
+        if colormode == COLOR_MODE_BGR and not hasattr(lv.color_t().ch, 'green_l'):
+            raise RuntimeError('st7789 BGR color mode requires defining LV_COLOR_16_SWAP=1')
+
+        self.display_name = 'ST7789'
+        self.display_type = DISPLAY_TYPE_ST7789
+
+        # MADCTL[rot % 4]
+        madctl = [0x00, 0x60, 0xc0, 0xa0][rot % 4]
+
+        #  width, height, xstart, ystart)[rot % 4]
+        if 320 in [width, height]:
```

Comment:
`320 in [width, height]` would return true for width=320, height=1000, for example.  
If your intent is to require only specific resolutions, why not simply check for them?
```python
if [width, height] == [320, 240]:
```

Reply:
That makes sense. Will do.